File: php/example/example1.php
<?php

include __DIR__.'/../vendor/autoload.php';

use Vittee\ThaiNum\ThaiNum;

var_dump(ThaiNum::text(1234.00002));
var_dump(ThaiNum::bahtText(1234.21000000));
var_dump(ThaiNum::bahtText(-6974321.756890000));

File: java/src/com/vittee/thainum/example/Example1.java
package com.vittee.thainum.example;

import com.vittee.thainum.ThaiNum;

public class Example1 {
    public static void main(String[] args) {
        ThaiNum thaiNum = new ThaiNum();

        System.out.println(thaiNum.text(1234.00002));

        double[] values = new double[] { 121.25650, -51.595013, 111.80 };
        for (double v : values) {
            System.out.println(thaiNum.bahtText(v));
        }
    }
}
